modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/DartDioClientCodegen.java

```diff
@@ -289,6 +289,7 @@ public void postProcessModelProperty(CodegenModel model, CodegenProperty propert
         Map<String, Object> operations = (Map<String, Object>) objs.get("operations");
         List<CodegenOperation> operationList = (List<CodegenOperation>) operations.get("operation");
 
+        Set<Map<String, Object>> serializers = new HashSet<>();
```

Comment:
are you sure this needs to be a set of maps? why not something simpler to debug like a list of tuples?

Reply:
I am not sure what mustache supports here, I assume with pairs you would somehow have to access `.first` `.second` in the templates, never seen this.